JASP-Desktop/components/JASP/Widgets/AboutWindow.qml

```diff
@@ -0,0 +1,226 @@
+	}
+
+	Rectangle
+	{
+		id: jaspLogo
+
+		width: parent.height/3
+		height: parent.height/5.5
+		anchors.left: parent.left
+		anchors.leftMargin: 50
+		anchors.verticalCenter: parent.verticalCenter
+		Image {
+			anchors.fill: parent
+			fillMode: Image.PreserveAspectFit
+			source: "qrc:/core/img/jasp-logo-black.svg"
+		}
+	}
+
+	Rectangle{
+		id: aboutInfoBox
+
+		anchors.left: jaspLogo.right
+		anchors.leftMargin: 25
+		anchors.right: parent.right
+		anchors.rightMargin: 25
+		anchors.top:topWave.bottom
+		anchors.bottom:bottomWave.top
+		border.color: "gray"
+		border.width:0
+
+		Text {
+			id: copyrightMessage
+			text: aboutModel.jaspCopyrightMessage
+			anchors.left:parent.left
+			anchors.top:parent.top
+			anchors.right: parent.right
+			height:25
+			}
+
+		Label {
+			id:jaspVersionLabel
+
+			text:qsTr("<b>Version: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:copyrightMessage.bottom
+			height:25
+			width:75
+
+			Text {
+				id: jaspVersionText
+
+				text:aboutModel.jaspVersionString
+				anchors.left:parent.right
+				anchors.top:parent.top
+				height:25
+			}
+		}
+
+		Label {
+			id:buildDateLabel
+
+			text:qsTr("<b>Build on: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:jaspVersionLabel.bottom
+			height:25
+			width:75
+
+			Text {
+				id: buildDateText
+
+				text:aboutModel.jaspBuildDate
+				anchors.left:parent.right
+				anchors.top:parent.top
+				height:25
+			}
+		}
+
+		Label {
+			id:downloadLabel
+
+			text:qsTr("<b>Download: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:buildDateLabel.bottom
+			height:25
+			width:75
+
+			Text {
+				id: downloadText
+
+				text:"<a href=\"" + aboutModel.jaspDownloadURL +"\">"+ aboutModel.jaspDownloadURL + "</a>"
+				anchors.left:parent.right
+				anchors.top:parent.top
+				textFormat: Text.StyledText
+				height:25
+				MouseArea {
+					id:mouseAreaDownload
+					anchors.fill: parent
+					onClicked: {
+						Qt.openUrlExternally(aboutModel.jaspDownloadURL)
+					}
+					cursorShape: Qt.PointingHandCursor
+				}
+			}
+		}
+
+		Label {
+			id:citationLabel
+
+			text:qsTr("<b>Citation: </b>")
+			color: labelcolor
+			anchors.left:parent.left
+			anchors.top:downloadLabel.bottom
+			height:25
+			width:75
+
+			Text {
+				id: citationText
+
+				text:aboutModel.jaspCitation
+				anchors.left:parent.right
+				anchors.top:parent.top
+				height:15
+			}
+
+			Text {
+				id: bibTexText
+
+				text:"<a href=\"https://jasp-stats.org/faq/#citation\">(BibTex)</a>"
+				anchors.left:citationLabel.right
+				anchors.top:citationText.bottom
+				textFormat: Text.StyledText
+				height:15
+
+				MouseArea {
+					id:mouseAreaBibTex
+
+					anchors.fill: parent
+					onClicked: {
+						Qt.openUrlExternally("https://jasp-stats.org/faq/#citation")
+					}
+					cursorShape: Qt.PointingHandCursor
+				}
+			}
+		}
+
+		Text {
+			id: warrantyText
```

Comment:
Can we grey this out a little like in the old About window? I don't think people's focus needs to go here immediately.